indico/web/client/js/react/components/DatePicker.jsx

```diff
@@ -9,24 +9,29 @@ import moment from 'moment';
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import {FinalField, validators as v} from 'indico/react/forms';
 import {Param, Translate} from 'indico/react/i18n';
 import {formatDate} from 'indico/utils/date_format';
 import {fromISOLocalDate} from 'indico/utils/date_parser';
 
 import 'indico/custom_elements/ind_date_picker';
 
+const INVALID = '__invalid__';
```

Comment:
Symbol is another option, but no strong preference
```suggestion
const INVALID = Symbol('invalid');
```

Reply:
nope, doesn't work, the ind-datepicker crashes when receiving a symbol instead of a string as its value